packages/sites/src/drafts/_get-json-resource.ts

```diff
@@ -0,0 +1,21 @@
+import { IHubRequestOptions } from "@esri/hub-common";
+import { getPortalUrl } from "@esri/arcgis-rest-portal";
+import { request, IRequestOptions } from "@esri/arcgis-rest-request";
+
+/**
+ * Dog-eared for rest-js
+ * @param {*} itemId
+ * @param {*} resourceName
+ * @param {*} requestOptions
+ */
+export function _getJsonResource(
+  itemId: string,
+  resourceName: string,
+  requestOptions: IRequestOptions
+) {
+  const url = `${getPortalUrl(
+    requestOptions
+  )}/content/items/${itemId}/resources/${resourceName}`;
+  const options = Object.assign({ params: { f: "json" } }, requestOptions);
```

Comment:
I'm not sure `f=json` is really a thing on this? i.e. this loads https://dc.mapsqa.arcgis.com/sharing/rest/content/items/a9e4560a26b848e899a0bbff890b1a4a/resources/draft-1598396441045.json

Reply:
It's to tell the `request` function how to parse the response. Ref: https://github.com/Esri/arcgis-rest-js/blob/master/packages/arcgis-rest-request/src/request.ts#L370

Reply:
👍